package.json

```diff
@@ -38,6 +41,7 @@
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "@typescript-eslint/parser": "^5.43.0",
     "@vitejs/plugin-react": "^2.2.0",
+    "dotenv": "^16.0.3",
```

Comment:
this is still inside dependencies. Should be inside devDependencies

Reply:
![image](https://user-images.githubusercontent.com/72614768/219842751-898d6d99-2b99-4404-a847-0bfb87ba9358.png)
Both of them are in devDependencies now.